src/main/common/maths.h

```diff
@@ -34,6 +34,8 @@
 
 // Use floating point M_PI instead explicitly.
 #define M_PIf       3.14159265358979323846f
+#define M_TWO_PIf   2.0f * M_PIf
```

Comment:
Always use parentheses around macro. 

`2 /  M_TWO_PIf` expands to `2 /  2.0f * M_PIf` (== M_PIf). This is extremely dangerous and hard to find bug.

Reply:
That's good to know!  Thank you!